tokens/eth/0x679131F591B4f369acB8cd8c51E68596806c3916.json

```
@@ -0,0 +1,33 @@
+{
+  "symbol": "TLN",
+  "address": "0x679131F591B4f369acB8cd8c51E68596806c3916",
+  "decimals": "18",
```

decimals must be a number
```suggestion
  "decimals": 18,
```